index.bs

```diff
@@ -218,7 +218,7 @@ following steps:
  <li>If |request|'s [=request/client=] is not null, then for each [=client hints token=] |requestHint| in
 |request|'s [=environment settings object/client hints set=], [=set/append=] |requestHint| to
 |hintSet|.
- <li>For each |possibleHint| in |hintSet|, if |request| is a [=subresource request=] and the result of
+ <li>For each |possibleHint| in |hintSet|, if |request| is a [=subresource request=] or a [=non subresource request=] (except "sharedworker" and "worker") and the result of
```

Comment:
This reads slightly confusing to me - is the point "requests for anything except (some, what about ServiceWorker?) workers"? 

Also, is [=non subresource request=] defined anywhere?

Reply:
The link is fixed now (https://fetch.spec.whatwg.org/#non-subresource-request) the category of navigation request is distinct and also not included.

ServiceWorkers support client hints while Workers and SharedWorkers do not. My understanding is that that it's because ServiceWorkers are short lived network proxies which can inherit the permissions policy of the creator, while Workers and SharedWorkers are longer lived (or shared across contexts) and have no permissions policy to inherit (thus, no way to limit client hint sharing).